scipy/linalg/matfuncs.py

```diff
@@ -668,3 +672,191 @@ def rounded_sign(x):
         return S0
     else:
         return S0, errest
+
+
+def is_matrix_symmetric(A, rtol=1e-05, atol=1e-08):
```

Comment:
These helper functions won't fly in their current state. Especially with hard coded tolerances. Also hermitianness is needed here. Real versions are not enough.

Reply:
Added hermitianness and symmetry tested with a new method.